frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/shareModal/shareModal.component.tsx

```diff
@@ -18,50 +18,44 @@
 import { FormModal } from '@controls/modal/formModal/formDialog.component';
 import { formatMessage } from '@/v5/services/intl';
 import { FormattedMessage } from 'react-intl';
-import { viewerRoute } from '@/v5/services/routing/routing';
-import { useParams } from 'react-router-dom';
 import { ShareTextField } from '@controls/shareTextField';
-import { IContainer } from '@/v5/store/containers/containers.types';
-import { IFederation } from '@/v5/store/federations/federations.types';
-import { DashboardParams } from '@/v5/ui/routes/routes.constants';
 import { MailToButton } from './shareModal.styles';
 
 type IShareModal = {
-	openState: boolean;
+	// the title in the modal
 	title: string;
-	containerOrFederation: IContainer | IFederation;
+	// the title in the email
+	openState: boolean;
+	name: string;
+	link: string;
 	onClickClose: () => void;
 };
 
 export const ShareModal = ({
 	openState,
 	title,
-	containerOrFederation,
+	name,
 	onClickClose,
-}: IShareModal): JSX.Element => {
-	const { teamspace, project } = useParams<DashboardParams>();
-	const link = viewerRoute(teamspace, project, containerOrFederation, null, true);
-
-	return (
-		<FormModal
-			open={openState}
-			onClickClose={onClickClose}
-			title={title}
-			showButtons={false}
-		>
-			<ShareTextField
-				label={formatMessage({
-					id: 'shareModal.linkLabel',
-					defaultMessage: 'Link',
-				})}
-				value={link}
+	link,
+}: IShareModal) => (
+	<FormModal
+		open={openState}
+		onClickClose={onClickClose}
+		title={title}
+		showButtons={false}
+	>
+		<ShareTextField
+			label={formatMessage({
+				id: 'shareModal.linkLabel',
+				defaultMessage: 'Link',
+			})}
+			value={link}
+		/>
+		<MailToButton href={`mailto:?subject=3D Repo container - ${name}&body=${link}`}>
```

Comment:
The mail to subject says 'container' even though it might be a federation or projct

Reply:
Good one, I fixed it